requirements.txt

```diff
@@ -11,5 +11,7 @@ boto3==1.9.138
 requests==2.21.0
 addict==2.2.1
 yamllint>=1.15.0
+rfc3987==1.3.8
```

Comment:
where is this used?

Reply:
@ramaro 
This is used to validate the source against valid rfc protocol in json schema.